phoenix-spark/src/it/resources/tenantSetup.sql

```diff
@@ -15,3 +15,4 @@
 -- limitations under the License.
 
 CREATE VIEW IF NOT EXISTS TENANT_VIEW(TENANT_ONLY_COL VARCHAR) AS SELECT * FROM MULTITENANT_TEST_TABLE
+UPSERT INTO TENANT_VIEW (ORGANIZATION_ID, TENANT_ONLY_COL) VALUES ('defaultOrg', 'defaultData')
```

Comment:
Adding some default data so we can know something was actually read in the "read" tests